pkg/virt-handler/vm.go

```diff
@@ -187,7 +189,51 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 		return err
 	}
 
-	controller.NewVirtualMachineInstanceConditionManager().CheckFailure(vmi, syncError, "Synchronizing with the Domain failed.")
+	// Update the condition when GA is connected
+	// channelState is 0 : nothing, 1: channel present, 2: channel connected
+	channelState := 0
+	channelNotPresent := true
+	if domain != nil {
+		for _, channel := range domain.Spec.Devices.Channels {
+			if channel.Target != nil {
+				log.Log.V(4).Infof("Channel: %s, %s", channel.Target.Name, channel.Target.State)
+				if channel.Target.Name == "org.qemu.guest_agent.0" {
+					channelNotPresent = false
+					if channel.Target.State == "disconnected" {
```

Comment:
Can we have constants for `disconnected` and `connected`? The best place would probably be `pkg/virt-launcher/virtwrap/api/schema.go`.

Reply:
Constants make sense, will add them.